floodviz/hydrograph_utils.py

```diff
@@ -60,9 +67,21 @@ def req_hydrodata(sites, start_date, end_date, url_top):
         returns a dictonary with the requested data from the nwis service 
```

Comment:
Okay, this may warrant a little more explanation then so the user of the function knows what is returned. But it is a list that is returned even if it's a single element.